zk/tests/test_zk.py

```diff
@@ -66,3 +67,22 @@ def test_error_state(aggregator, dd_environment, get_conn_failure_config):
     expected_mode = get_conn_failure_config['expected_mode']
     mname = "zookeeper.instances.{}".format(expected_mode)
     aggregator.assert_metric(mname, value=1, count=1)
+
+
+@pytest.mark.usefixtures('dd_environment')
+def test_metadata(check, get_instance, aggregator, datadog_agent):
```

Comment:
```suggestion
def test_metadata(check, get_instance, version_metadata, datadog_agent):
```

No need to redefine `version_metadata` as you already created a fixture for it.
Also aggregator was not used